src/main/python/mlsearch/api_requester.py

```diff
@@ -161,7 +173,58 @@ def _fetch_paperwithcode(self) -> [Protocol]:
             self.data['content'] = [proto.to_JSON() for proto in results]
 
         self.data['response_code'] = query_result.status_code
-        
+    
+    def _fetch_youtube(self, next_page_token=None) -> [Protocol]:
+        """Fetch the Youtube Repository"""
+        results = []
+        youtube = googleapiclient.discovery.build(
+            self._config.YOUTUBE_SERVICE_NAME, 
+            self._config.YOUTUBE_API_VERSION, 
+            developerKey = self._config.YOUTUBE_DEVELOPER_KEY)
```

Comment:
I think there's a chance that the developer key is empty. I think we should handle that kind of error?

Reply:
Technically the default `MLT account` will be used instead if the `null` input is assigned from the client. Moreover, if the developer key is empty or null, then the lambda will throw an error back to the client.

Reply:
Thanks, Sai. I mean, since we already know there's a potential error thrown by lambda, I think it may be hard to debug when it happens because the error comes from lambda. If we take care of that error, we may know the problem faster.
But since
> the default MLT account will be used instead if the null input is assigned from the client

I think it'll be fine.